app/concepts/reservations/use_cases/create_offline.rb

```diff
@@ -12,6 +12,7 @@ def call
         ActiveRecord::Base.transaction do
           repository.create!(reservation_params).tap do |reservation|
             Tickets::UseCases::Create.new(reservation: reservation, tickets: params[:tickets]).call
+            cancel_expired(reservation: reservation)
```

Comment:
I think this is redundant for offline tickets because we assume clients pay for tickets buying them at the cinema 🤔

Reply:
I'm still not sure 🙈 after we all talked yesterday I think that there are two groups of people - once allows to reservation tickets at offline desk and second don't. So I've decided to let my clients do this (for now, maybe I will delete it) But thanks for catching it anyway ❤️